test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -111,6 +111,15 @@ items:
     - get
     - list
     - watch
+  - apiGroups:
```

Comment:
actually interesting that we make these world-readible. Knowing the admission webhook services might give important info for a possible attack.

Reply:
> actually interesting that we make these world-readible. Knowing the admission webhook services might give important info for a possible attack.

`cluster-reader` is a read-only cluster-admin who can see everything in the cluster but secrets and oauth tokens.  Very few have this power.